sql_generators/mobile_kpi_support_metrics/templates/active_users.view.sql

```diff
@@ -61,7 +61,7 @@ SELECT
       CAST(NULL AS STRING)
   END AS device_type,
   EXTRACT(YEAR FROM first_seen_date) AS first_seen_year,
-  app_display_version AS app_version,
+  NULLIF(app_display_version, "") AS app_version,
```

Comment:
When looking at the data noticed "" values. Doing this to make sure absence of value is represented with a `null`